### ISSUE TITLE
lein test `show difference` dialog disappears when trying to access it

### ISSUE THREAD

**Issue by stopachka** [cursiveclojure/cursive#1013]
when I run `Run test in current NS in Repl`, and there's an error, Cursive shows me where it is (so awesome).

On hover, I see a dialog screen, that when I click show difference can help me make sense of it.

However, when trying top have that dialog open is a nightmare -- as soon as I move my mouse to  get on the dialog, it disappears.

Has anyone dealt with this before?

Cursive 0.1.57
Intellij IDEA 14.1.4

video -> https://dl.dropboxusercontent.com/u/4188273/cursive-show-difff.mov

**Comment by danielcompton** [comment]
I had the same thoughts today when running some test.

**Comment by jarppe** [comment]
Same issue. I used to be able to open the dialog that shows the difference, but it seems that functionality is no longer available.

Is there any way to open the difference dialog anymore?

IntelliJ IDEA 2016.2.2
Cursive 1.4.0-eap2-2016.2

**Comment by cursive-ide** [comment]
Yes, one thing that works (assuming the test result gutter icon only refers to a single test and not multiple executions) is clicking the gutter icon - that's the most reliable way to see the diff.
